ruby/vampires.rb

```diff
@@ -0,0 +1,43 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+until employees == 0
+	puts "What is your name?"
+	name = gets.chomp.downcase
+	puts "How old are you? What year were you born?"
+	age = gets.chomp
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+	garlic_bread = gets.chomp
+	puts "Would you like to enroll in the company's health insurance?(y/n)"
+	health_insurance = gets.chomp
+
+	if (age.to_i < 120) && (garlic_bread == 'y' || health_insurance == 'y')

```

Comment:
Age should be checking against the birth year and the age given.  Not 120 years old.  Please recheck the directions and consider using `Time.now.year` for a dynamic result.